lldb/source/Target/SwiftLanguageRuntimeImpl.h

```diff
@@ -78,7 +78,8 @@ class SwiftLanguageRuntimeImpl {
 
   /// Ask Remote Mirrors for the type info about a Swift type.
   const swift::reflection::TypeInfo *
-  GetTypeInfo(CompilerType type, ExecutionContextScope *exe_scope);
+  GetTypeInfo(CompilerType type, ExecutionContextScope *exe_scope,
+              swift::reflection::TypeRef const **out_tr = nullptr);
```

Comment:
Added an out parameter to return the typeref.

Reply:
It might be cleaner to factor out a function

`const swift::reflection::TypeInfo *GetTypeInfo(swift::reflection::TypeRef type_ref)`

and call that directly where we need the type_ref? If it isn't, then returning either a `std::pair<>` or a `struct {TypeInfo type_info; TypeRef type_ref}` may be more elegant.